### ISSUE TITLE
SCSS/Less variable color change

### ISSUE THREAD

**Issue by muuvmuuv** [Selz/plyr#575]
It would be a great features if you would include the main style after the variables in a separate file, so I could change some variables by myself and compile it to my project. Now I have to copy all the plyr.scss stuff and paste the custom variable after the varibles import.

**Comment by SamPotts** [comment]
Sure - that'd be a simple change to make

**Comment by thasmo** [comment]
Indeed, that would be a really useful change! At present the less styles can't be used in a nice way because it enforces to include the plyr less-file before the project variables.

**Comment by flowen** [comment]
would love to include the scss files from the node_modules and be able to overwrite the variables with my own. Right now I copy the plyr.scss to avoid the imports. I would have to manually copy the plyr.scss for every update.
